spec/models/elastic_tweet_spec.rb

```diff
@@ -15,7 +15,7 @@
     ElasticTweet.commit
   end
 
-  describe ".search_for" do
+  pending ".search_for" do
```

Comment:
Can you see if these specs pass now? Also, if these still need to be disabled, please add a comment re. the Twitter deprecation.

Reply:
Ok so, back then we decided to comment out because elastic_tweet is dead code and rspec was failing but at this point discovered analyze is also causing tests cases to fail which may be the result of this one as well. So placing it back to describe for now till we get analyze set to standard. Then after we will see if rspec still fails and decide from there.